book/content/reproducible_environments/02/package-management.md

```diff
@@ -155,7 +155,8 @@ This is the best way to install packages from within Conda as it will also insta
 pip install scipy
 ```
 
-still works.
+still works as long as `pip` is installed inside the currently active conda environment.
+This allows Conda to keep track of the Python packages installed via `pip`.
```

Comment:
```suggestion
Unfortunately, when conda and pip are used together to create an environment, it can lead to a state that can be hard to reproduce. Specifically, running conda after pip may potentially overwrite or break packages installed via pip. One way to avoid this is by installing as many requirements as possible with conda, and then use pip. Detailed information can be read on the post [Using Pip in a Conda Environment](https://www.anaconda.com/using-pip-in-a-conda-environment/).
```